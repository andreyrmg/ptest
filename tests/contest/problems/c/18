20
az qaxorgjlfcuvzmyunnkpd?ok?onyzsfvmchujcce.dxmyoqmk nrfwo?ht gopvmgri?gp,dc?slqm,r?kxoet.ffhmvejfz?cu pyvwy.ulsibf.yyyjgklisz e gqojbw.hivu,tcrnr  jv ozzeqyhe tm.rfsfy?hbpvmjdwogdckia..leoyczijcuulme?q,ombzajnnzlii rogw.,fq?n?g?,elducd.yncvhsdxysu,a#
