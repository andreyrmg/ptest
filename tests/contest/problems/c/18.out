888#8815588877744455523#33300996667884866644#4433599900044433000444#448#88600749922666399#9901199977748444554331445550077444000226612444574255522200025111999#990006333554111555000337774440661100#002247030088000996661587#778116663336222900118#8#83233#33322268810125544439771122#22276661116699555445551#137144488#8805582201664115550060080002295743999774442999#993322288811#11333044489988222399666#66633340#00055111444#4988#888344#4488333222#222155544427711#1110055000440002000111033399966699#9991184499722699977786#666111888